app/main.py

```diff
@@ -16,4 +16,15 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+all_data = [lucky_number, pi, one_is_a_prime_number, name, my_favourite_films, profile_info, marks, collection_of_coins]
+mut, immut = [], []
+
+for el in all_data:
+    if type(el) == int or type(el) == float or type(el) == str or type(el) == tuple or type(el) == bool:
+        immut.append(el)
+    else:
+        mut.append(el)
+sorted_variables = {
+    "mutable": mut,
+    "immutable": immut
+}
```

Comment:
just place related variables to the related fields

Reply:
at now is correct?